### ISSUE TITLE
[UST-138] [Frontend] Claim reputation in the background

### ISSUE THREAD

**Issue by UranusLin** [social-tw/social-tw-website#449]
## Description

- Execute `API` :: `Fetch all waiting-for-transaction reports`
- Add `API` :: `ClaimPositvieReputation`
- Add `API` :: `ClaimNegativeReputation`
- Add `Proof` :: `Positive Reputation Proof`
- Add `Proof` :: `Negative Reputation Proof`
- Check to claim reputation of reports one by one

## Test

- Fetch all waiting-for-transaction reports
- Claim positive reputation in the background
- Claim negative reputation in the background

**Comment by UranusLin** [comment]
The endpoint for fetching all waiting-for-transaction reports is already in the main branch:
```
GET /api/report?status=${ReportStatus.WAITING_FOR_TRANSACTION}&publicSignals=${epochKeyLitePublicSignals}&proof=${epochKeyLiteProof}
```
The single reputation claim API is included in [PR #440](https://github.com/social-tw/social-tw-website/pull/440).

To claim all reputations at once, a new API should be created. The request should include:

For reporters:
```json
[
  { "reportId": "...", "epochKey": "...", "epoch": "..." },
  { "reportId": "...", "epochKey": "...", "epoch": "..." }
]
```
For posters:
```json
[
  { "reportId": "...", "epochKey": "...", "epoch": "..." },
  { "reportId": "...", "epochKey": "...", "epoch": "..." }
]
```
For voters:
```json
[
  { "reportId": "...", "nullifier": "...", "epoch": "..." },
  { "reportId": "...", "nullifier": "...", "epoch": "..." }
]
```

**Comment by cc03668** [comment]
Arthur：如果傳一串 list 表示使用者要生成很多很多證明再傳給後端，那會不會造成時間太久、time out？
